ci/infra/testrunner/tests/test_skuba_upgrade.py

```diff
@@ -133,17 +133,21 @@ def test_upgrade_apply_user_lock(setup, platform, kubectl, skuba):
     setup_kubernetes_version(skuba, PREVIOUS_VERSION)
 
     # lock kured
-    kubectl.run_kubectl("-n kube-system annotate ds kured weave.works/kured-node-lock='{\"nodeID\":\"manual\"}'")
+    kubectl.run_kubectl(
+        "-n kube-system annotate ds kured weave.works/kured-node-lock='{\"nodeID\":\"manual\"}'")
 
     outs = {}
     for (r, n) in [("master", 0), ("worker", 0)]:
```

Comment:
@danielorf here e.g. the numbers should be applied from a CONSTANT, e.g. `MASTER_NUM` and `WORKER_NUM`

otherwise we'll only upgrade one worker and one master

Reply:
Event better, you can retrieve the number of masters and workers using skuba: 
https://github.com/SUSE/skuba/blob/master/ci/infra/testrunner/tests/conftest.py#L41-L48